pnpm-lock.yaml

```diff
@@ -9251,4 +9251,4 @@ packages:
 
   /zwitch@2.0.4:
     resolution: {integrity: sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==}
-    dev: false
+    dev: false
```

Comment:
I guess this change is unnecessary, right?

Reply:
It looks identical to me. It's caused by me pulling back changes on [pnpm-lock.yaml](https://github.com/apache/yunikorn-site/pull/451/commits/8989cd5a598965b08b46c95bb2eceec065e378de#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bb) on my first commit(caused by normal pnpm compilation). Also, wanna ask should I preserve those changes caused by normal compilation?

Reply:
normally, we should avoid unnecessary changes. Hence, please revert it

Reply:
Done.